public/controllers/agent/components/agents-preview.js

```diff
@@ -120,15 +120,15 @@ export const AgentsPreview = compose(
         this.setState({ platforms: platformsModel, loading: false });
     } catch (error) {}
   }
-  
+
   removeFilters(){
     this._isMount && this.setState({agentTableFilters: []})
   }
 
   render() {
     const colors = ['#017D73', '#bd271e', '#69707D'];
     return (
-      <EuiPage>
+      <EuiPage style={{ flexDirection: 'column'}}>
```

Comment:
Could we put this in a CSS class?

Reply:
Yes, it is possible to make that change, thank you.